weldx/tests/test_measurement.py

```diff
@@ -1,112 +1,369 @@
-        error=msm.Error(2.0),
-    )
+        """
+        default_kwargs = dict(
+            name="source", output_signal=Signal("analog", "V"), error=Error(0.01)
+        )
 
-    dp_04 = msm.DataTransformation(
-        name="Calibration voltage measurement",
-        input_signal=dp_03.output_signal,
-        output_signal=msm.Signal("digital", "V", data=data_02),
-        error=msm.Error(Q_(3.0, "percent")),
-    )
+        if kwargs is not None:
+            default_kwargs.update(kwargs)
 
-    chn_01 = msm.MeasurementChain(
-        name="Current measurement", data_source=src_01, data_processors=[dp_01, dp_02]
-    )
+        return default_kwargs
+
+    @classmethod
+    def _default_init_kwargs(
+        cls, kwargs: Dict = None, source_kwargs: Dict = None
+    ) -> Dict:
+        """Return a dictionary of keyword arguments required by the `__init__` method.
+
+        Parameters
+        ----------
+        kwargs :
+            A dictionary containing some key word arguments that should replace the
+            default ones.
+
+        Returns
+        -------
+        Dict :
+            Dictionary with keyword arguments for the `__init__` method
+
+        """
+        source_kwargs = cls._default_source_kwargs(source_kwargs)
+
+        default_kwargs = dict(
+            name="name",
+            source=SignalSource(**source_kwargs),
+            signal_data=[1, 3, 5],
+        )
+        if kwargs is not None:
+            default_kwargs.update(kwargs)
+
+        return default_kwargs
+
+    @staticmethod
+    def _default_transformation(kwargs: Dict = None) -> SignalTransformation:
+        """Return a default `SignalTransformation`.
+
+        Parameters
+        ----------
+        kwargs :
+            A dictionary containing some key word arguments that should replace the
+            default ones.
 
-    chn_02 = msm.MeasurementChain(
-        name="Voltage measurement", data_source=src_02, data_processors=[dp_03, dp_04]
+        Returns
+        -------
+        Dict :
+            Default `SignalTransformation`
+
+        """
+        default_kwargs = dict(
+            name="transformation",
+            error=Error(0.1),
+            func=MathematicalExpression("a*x", parameters={"a": Q_(1, "1/V")}),
+            type_tf="AD",
+        )
+        if kwargs is not None:
+            default_kwargs.update(kwargs)
+
+        return SignalTransformation(**default_kwargs)
+
+    @classmethod
+    def _default_add_transformation_kwargs(cls, kwargs: Dict = None) -> Dict:
+        """Return a dictionary of keyword arguments required by `add_transformation`.
+
+        Parameters
+        ----------
+        kwargs :
+            A dictionary containing some keyword arguments that should replace the
+            default ones.
+
+        Returns
+        -------
+        Dict :
+            Dictionary with keyword arguments for the `add_transformation` method
+
+        """
```

Comment:
```suggestion
    def _default_add_transformation_kwargs(cls, replace_kwargs: Dict = None) -> Dict:
        """Return an updated dictionary of default keyword arguments required by `add_transformation`."""
```
this makes it much easier to scroll through the code :)